insecure/net/network_ingress_test.go

```diff
@@ -0,0 +1,96 @@
+package net
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/engine/testutil"
+	"github.com/onflow/flow-go/insecure"
+	mockinsecure "github.com/onflow/flow-go/insecure/mock"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/libp2p/message"
+	"github.com/onflow/flow-go/network/channels"
+	"github.com/onflow/flow-go/network/mocknetwork"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// TestHandleIncomingEvent_AttackerRegistered checks that a corrupt network sends ingress messages to a registered attacker.
+// The attacker is mocked out in this test.
+func TestHandleIncomingEvent_AttackerRegistered(t *testing.T) {
+	codec := unittest.NetworkCodec()
+	corruptedIdentity := unittest.IdentityFixture(unittest.WithAddress(insecure.DefaultAddress))
+	flowNetwork := &mocknetwork.Network{}
+	ccf := &mockinsecure.CorruptConduitFactory{}
```

Comment:
```suggestion
	ccf := mockinsecure.NewCorruptConduitFactory(t)
```

Reply:
replaced a lot of places to use mock constructor https://github.com/onflow/flow-go/pull/3255/commits/e8d7b5f3620e7a7a8070ec9a1ca1949b8e22c3d6